findmnt: update page (#14614)
tldr-pages/tldr: pages/linux/findmnt.md

- Find filesystems with specific label:

`findmnt LABEL={{BigStorage}}`

> - Check mount table content with detail (default is to verify `/etc/fstab`):

Reviewer: ```suggestion
- Check mount table content in detail and verify `/etc/fstab`:
```
